LocalSettings.php

```diff
@@ -3096,6 +3096,10 @@
 			'member',
 		],
 		'+hypopediawiki' => [
+			'',
```

Comment:
This won't work right, I don't think. This would basically. Be duplication of them so would be `$wgRestrictionLevels` = [ '', 'user', 'autoconfirmed', 'sysop', '', 'user', autoconfirmed', 'sysop', 'bureaucrat' ];`